host/host.go

```diff
@@ -38,6 +38,7 @@ type Frame struct {
 	File          FileID
 	Lineno        libpf.AddressOrLineno
 	Type          libpf.FrameType
+	Extra         libpf.AddressOrLineno
```

Comment:
While I see the benefit of passing additional information, e.g. it could also help improving the use of error frames, the naming `Extra` feels not right to me. As naming is always hard, I don't have a proper alternative that would describe the use case in a better way.